[#508] [FEATURE] Récupérer depuis l'api les profiles partagés avec une organisation donnée (US-734)

florianEnoh opened the pull request.

Commit:
add a service to compute completion percentation

Commit:
add a service to compute completion percentation

Commit:
Add new column completion percentage to snapshots table

Commit:
Add new column completion percentage to snapshots table

Commit:
Add profile completion service$

Commit:
Add profile completion service$

Commit:
Add Profile completion service to snapshot controller

Commit:
Add Profile completion service to snapshot controller

Commit:
Add new arg ProfileCompetion to snapshot service#create

Commit:
Add new arg ProfileCompetion to snapshot service#create

Commit:
Add new endpoint /api/organizations/:id/snapshots

Commit:
Add new endpoint /api/organizations/:id/snapshots

Commit:
implement getSharedProfiles in organizationController

Commit:
implement getSharedProfiles in organizationController

Commit:
Add snapshot serializer

Commit:
Add snapshot serializer

Commit:
Add new unit test for snapshot controller#getSharedProfiles

Commit:
Add new unit test for snapshot controller#getSharedProfiles

Commit:
Add new tested repository method#getSnapshotsByOrganizationId

Commit:
Add new tested repository method#getSnapshotsByOrganizationId

Commit:
Try to serialize

Commit:
Try to serialize

Commit:
Add method serializeArray to snapshot-serializer

Commit:
Add method serializeArray to snapshot-serializer

Commit:
add relation eager loading in getSnapshotsByOrganizationId (snapshotRepository)

Commit:
add relation eager loading in getSnapshotsByOrganizationId (snapshotRepository)

Commit:
Add convertionWithRelation

Commit:
Add convertionWithRelation

Commit:
Fix organizations snapshots url

Commit:
Fix organizations snapshots url

Commit:
Add user relations to snapshot model

Commit:
Add user relations to snapshot model

Commit:
Simplify snapshot repository

Commit:
Simplify snapshot repository

Commit:
Fix snapshot repository

Commit:
Fix snapshot repository

Commit:
Improve organizations#getSharedProfiles method

Commit:
Improve organizations#getSharedProfiles method

Commit:
Fix all tests relative to organizations controller getSharedProfiles

Commit:
Fix all tests relative to organizations controller getSharedProfiles

Commit:
Fix property name. CreattionDate to createdAt

Commit:
Fix property name. CreattionDate to createdAt

Commit:
Fix acceptance test - organizations controller get shared Profiles by adding createdAt

Commit:
Fix acceptance test - organizations controller get shared Profiles by adding createdAt

mackwic committed:
[#509] [FEATURE] Ajouter les endpoints /metrics et /errors/500 à l'API (US-723). (#509)

*  Add prometheus metrics

* [FEATURE] Add /metrics and /errors/500 endpoints

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Modify live/scripts/deploy.sh script

jbuget committed:
Merge branch 'fix-deploy-script' into dev

Commit:
Add little change to handle api deploiement

Commit:
Add little change to handle api deploiement

jbuget commented:
I've deployed this PR to http://594-show-shared-snapshots-for-logged-prescriber.pix.beta.gouv.fr. Please check it out

cmalaboeuf committed:
[#503] [FEATURE] Accepter les caractères spéciaux dans le mot de passe (US-663). (#503)

* Add space in validator for password

* Add support of all character in regex, front and back verification

* Add test as documentation:
A password must contain 8 characters, at least one letter and at least a figure

* Take into account diacritics (accents) and special caracters in password validation

* Remove useless args parameters in password validation (back)

jbuget committed:
[#504] Forcer le rechargement du profil à chaque fois qu'on accède à la page /compte (US-700) (#504)

* Force user profile data reload on account page

* Add a unit test on routes/compte-test

Commit:
Fix bug when score is null

Commit:
Fix bug when score is null

twickham committed:
[BUGFIX] Git commit should be spelled entirely

twickham committed:
[RELEASE] A minor is being released from 1.19.0 to 1.20.0.

jbuget commented:
I've deployed this PR to http://594-show-shared-snapshots-for-logged-prescriber.pix.beta.gouv.fr. Please check it out

Akhilian committed:
Update CHANGELOG.md

Akhilian committed:
Merge branch 'release-1.20.0' into dev

Commit:
bug fix, replace fetch by fetch all and create new method i, bookshelf utils

Commit:
bug fix, replace fetch by fetch all and create new method i, bookshelf utils

jbuget committed:
Lighten competence level bar color (#506)

jbuget committed:
[#510] Ajouter un loader mouf-mouf au démarrage d'un test (US-698). (#510)

* Add a loader during assessment creation (course loading).

* Remove Ember test and code coverage (tmp)

* Rename Interwind.gif into interwind.gif

* Bump Ember resolver (to fix build)

* Reactivate Code Coverage and Ember tests

* Revert "Reactivate Code Coverage and Ember tests"

This reverts commit bf0a26b33c025729061412e2fa36e6e38361f8aa.

jilljenn committed:
[#502] Intégrer la nouvelle règle de calcul des pix au scoring (US-671 & US-672) (#502)

* Introduce skillRepository

* Add getFromCompetence to challengeRepository

* Add skillRepository

* Add computePixScoreOfSkills to Course in CAT

* Remove scoring service and handle scoring in CAT

* Add obtainedLevel (US-672)

* Do not crash if challenge has no skill

* Fix tests, put SQLite 3.1.8, fix bug in previous PR regarding challengeSkills

cmalaboeuf committed:
[#505] [FEATURE] Ajout de la page "Mentions légales" (US-380). (#505)

* test

* Add legal notices page with links, images ...

* Fix pb :
- Remove comment
- refactor bem class
- add section
- fix typo
..

* tmp

* Add mobile support for page /mentions-legales

* Add a test on URL mapping

* Take into account remarks of Benjamin about logos

* Fix font-family error

* Remove useless class

jilljenn committed:
[#515] [BUGFIX] Ajout de la clé RECAPTCHA_KEY au fichier sample.env (#515)

[#515] [BUGFIX] Ajout de la clé RECAPTCHA_KEY au fichier sample.env

jilljenn committed:
[#513] [BUGFIX] Gestion du cache des repositories par compétence.

Commit:
Merge branch '594-show-shared-snapshots-for-logged-prescriber' of https://github.com/sgmap/pix into 594-show-shared-snapshots-for-logged-prescriber

jbuget committed:
Take into account code review remarks

jbuget committed:
BSR: extract magic number into const

jbuget commented:
I've deployed this PR to http://594-show-shared-snapshots-for-logged-prescriber.pix.beta.gouv.fr. Please check it out
